examples/telnet_server/Makefile

```diff
@@ -0,0 +1,50 @@
+# name of your application
+APPLICATION = telnet_server
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Include packages that pull up and auto-init the link layer.
+# NOTE: 6LoWPAN will be included if IEEE802.15.4 devices are present
+USEMODULE += netdev_default
+USEMODULE += auto_init_gnrc_netif
+# Activate ICMPv6 error messages
+USEMODULE += gnrc_icmpv6_error
+# Specify the mandatory networking modules for IPv6
+USEMODULE += gnrc_ipv6_default
+# Additional networking modules that can be dropped if not needed
+USEMODULE += gnrc_icmpv6_echo
+USEMODULE += netutils
+# Add also the shell, some shell commands
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+# Include the telnet server
+USEMODULE += stdio_telnet
+
+# Enable faster re-connects
+CFLAGS += -DCONFIG_GNRC_TCP_EXPERIMENTAL_DYN_MSL_EN=1
+
+# enable debug output via UART
+ifneq (native, $(BOARD))
+  FEATURES_OPTIONAL += periph_uart
+endif
```

Comment:
```suggestion
endif
FEATURES_BLACKLIST += arch_native
```

Reply:
with option 3. taken by benpicco i think this is resolved